src/Microsoft.AspNet.Mvc.Razor/RazorPage.cs

```diff
@@ -547,15 +547,6 @@ public virtual void WriteLiteralTo([NotNull] TextWriter writer, string value)
                 {
                     var attrVal = values[i];
                     var val = attrVal.Value;
```

Comment:
Remove these abbreviations while updating this method; perhaps `attributeValue` and `value`.